src/gsy_e/gsy_e_core/area_serializer.py

```diff
@@ -55,8 +56,10 @@ def default(self, o):
             return self._encode_leaf(o)
         if isinstance(o, BaseStrategy):
             return self._encode_subobject(o)
+        if isinstance(o, TradingStrategyBase):
```

Comment:
This is because the Heatpump strategies do not inherit from BaseStrategy right ? At any case, I agree with the addition!

Reply:
Exactly.